classes/class-wc-connect-taxjar-integration.php

```diff
@@ -315,15 +315,15 @@ public function _error( $message ) {
 		$invalid_postcode = false !== strpos( $formatted_message, 'isn\'t a valid postal code for' );
 		if ( ! is_admin() && ( $state_zip_mismatch || $invalid_postcode ) ) {
 			$fields = WC()->countries->get_address_fields();
-			$postcode_field_name = __( 'Postcode / ZIP', 'woocommerce-services' );
```

Comment:
This I wasn't sure if I had to modify as well 😬  Should I revert the changes on this file?

Reply:
I think this makes sense to update as well. Only thing I noticed was this change has spaces around the slash while the other instances does not have spaces.

Reply:
Yeah, the existing change had spaces around the slash - do you think this should be converted to _not_ have spaces?

Reply:
Yeah I think for consistency it shouldn't have spaces. Given the overall goal of the update, I'd put more weight on matching the new messaging over matching any previous messaging.

Reply:
Sure thing, that makes sense 👍  Done in 401a709